### PR TITLE
Adds clarifications from Trey.

### PR DESCRIPTION
**Pull request by johnnymatthews**
Replaces https://github.com/filecoin-project/filecoin-docs/pull/1911

### CODE REVIEW

File: content/en/storage-provider/filecoin-deals/verified-deals/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ When storing real customer data and not simply [CC sectors](https://docs.filecoi
 
 As a storage provider, you are a vital part of the ecosystem. Being a storage provider is not like being a miner on other blockchains - it requires much more than providing some disk space to the network. Making deals, and doing your own business development is vital to growing a business on Filecoin. You can get copies of data to store from other storage providers (as clients often store multiple copies with different providers, for redundancy), but even that requires you to network and build relationships with others. Bringing your own customers onto the Filecoin network requires people and processes for business development, sales and marketing, and might mean you have to hire additional sales people for your team.
```
Comment (ElPaisano) [inline content/en/storage-provider/filecoin-deals/verified-deals/index.md:30]: gpt + my suggested edits

rephrasing, removing redundant info, use bulleted list

```suggestion
As a storage provider, you play a crucial role in the ecosystem. Unlike miners in other blockchains, storage providers must do more than offer disk space to the network. Whether onboarding new customers to the network, or storing copies data from other storage providers for clients seeking redundancy, providing storage can involve:

- Business development.
- Sales and marketing efforts.
- Hiring additional personnel.
- Networking.
- Relationship building.
```

File: content/en/storage-provider/filecoin-deals/verified-deals/index.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ When storing real customer data and not simply [CC sectors](https://docs.filecoi
...
 
-Getting a copy of the data requires systems and infrastructure to allow for large volumes, sometimes up to <!--TODO STEF BOB can we give a meaningful example here--> of data to be ingested into your system. This requires substantial internet bandwidth, with a minimum of 10Gb. However, many large storage providers use up to 100Gb internet connections. As an example, transferring 1 PiB of data takes XX hours on a 10Gb connection.
+Getting a copy of the data requires systems and infrastructure to allow for large volumes, sometimes up to a PiB of data to be ingested into your system. This requires substantial internet bandwidth, with a minimum of 10Gb. However, many large storage providers use up to 100Gb internet connections. As an example, transferring 1 PiB of data takes ~240 hours on a 10Gb connection.
```
Comment (ElPaisano) [inline content/en/storage-provider/filecoin-deals/verified-deals/index.md:32]: gpt + my suggested edits

```suggestion
Acquiring data copies requires systems and infrastructure capable of ingesting large volumes of data, sometimes up to a PiB. This necessitates significant internet bandwidth, with a minimum of 10 Gb. For instance, transferring 1 PiB of data takes approximately 240 hours on a 10 Gb connection. However, many large storage providers use up to 100 Gb internet connections. ```

File: content/en/storage-provider/filecoin-deals/verified-deals/index.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ When storing real customer data and not simply [CC sectors](https://docs.filecoi
...
 
 Data preparation, which includes seperating files and folders in car files, takes time and skill. You can offload this task to a Data Preparer at a service cost, or you can choose to take on this role yourself. Tools like [Singularity](https://singularity.storage/) make this easier.
```
Comment (ElPaisano) [inline content/en/storage-provider/filecoin-deals/verified-deals/index.md:34]: CAR is an acronym so needs all caps
gpt + my edit

```suggestion
Data preparation, which involves separating files and folders in CAR files, is time-consuming and requires expertise. You can delegate this task to a Data Preparer for a fee or assume the role yourself. Tools like [Singularity](https://singularity.storage/) simplify this process.
```

File: content/en/storage-provider/filecoin-economics/fil-collateral/index.md [hunk 4]
```diff
@@ -19,11 +19,11 @@ As a storage provider on the network, you will have to create FIL wallets, and a
...
 
-When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), or when you take your storage offline and thereby break the rules of the storage deal, you will get penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited <!--TODO NOBLOCK STEF BOB where does it go? who gets it? Does it all go? -->and your storage power is reduced.
+When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), or when you take your storage offline and thereby break the rules of the storage deal, you will get penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited to the f099 address from your locked or available rewards and your storage power is reduced. The f099 address is the address in which all burned FIL go to.
```
Comment (ElPaisano) [inline content/en/storage-provider/filecoin-economics/fil-collateral/index.md:26]:
```suggestion
When a storage provider fails to answer to the WindowsPoSt challenges within the 30 minute deadline (see [Storage Proving]({{<relref storage-proving >}})), storage is taken offline, or any storage deal rules are broken, the provider is penalized against the provided collateral. This penalty is called [_slashing_]({{<relref slashing>}}) and means that a portion of the pledged collateral is forfeited to the `f099` address from your locked or available rewards and your storage power is reduced. The `f099` address is the address where all burned FIL goes.
```

Comment (ElPaisano) [reply content/en/storage-provider/filecoin-economics/fil-collateral/index.md:26]: @johnnymatthews should we use "Fil" or "FIL"?

Comment (johnnymatthews) [reply content/en/storage-provider/filecoin-economics/fil-collateral/index.md:26]: FIL.

**Pull request merged**